### DISCUSSION TITLE
Rewrite Status

### DISCUSSION THREAD

**Discussion by kzali** [multigcs/LinuxCNC-RIO#46] [General]
Hello Oliver
I was wondering as to the status of "rewrite."  Has development on that aborted or is it complete and is there a ptential release planned?
Thanks

**Comment by multigcs** [comment]
it's more or less finished, I'm just testing it

**Reply by etet100** [reply]
Let us test with you!

**Reply by kzali** [reply]
Happy to do that although I am terrible at it but will give it my best shot,  I may bug you with some silly questions!  I will be using my Tango board and Eth01 with an intel nuc.
I do the firmware on Ubuntu on a small laptop,  So no Raspberry PI.  will report back in a few days.
some wiring diagram samples for 3 tb6600 or similar drivers would be much appreciated

**Comment by multigcs** [comment]
https://github.com/multigcs/riocore
https://github.com/multigcs/riogui

**Comment by TurBoss** [comment]
hello, so its ok to add more boards?
